client/app/dedicated/server/bandwidth/order-agora/dedicated-server-bandwidth-agora-order.controller.js

```diff
@@ -0,0 +1,79 @@
+class ServerOrderAgoraBandwidthCtrl {
+  constructor($scope, $stateParams, $translate, User, Server) {
+    this.$scope = $scope;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.Server = Server;
+    this.User = User;
+    this.model = {};
+    this.data = {};
+    this.isLoading = false;
+    this.existingBandwidth = this.$scope.currentActionData.bandwidth.bandwidth.OvhToInternet.value;
+
+    this.steps = [
+      {
+        isValid: () => this.model.plan,
+        isLoading: () => this.isLoading,
+        load: () => {
+          this.isLoading = true;
+          return this.Server
+            .getBareMetalPublicBandwidthOptions(this.$stateParams.productId, this.existingBandwidth)
+            .then((res) => {
+              this.isLoading = false;
```

Comment:
```this.isLoading = false;``` should be in `finally`

Reply:
In `catch` block, the modal is closed so did not add finally